exe/bolt-server

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env ruby
-# frozen_string_literal: true
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
I think we should stick with looking up ruby via env. The gem install (or our service config) should take care of making sure it finds the right ruby.

Reply:
Philosophically, I don't think this file should know about specific install paths.